docs/javascript/v2/changelog/release-notes.mdx

```diff
@@ -14,6 +14,16 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
 
+## 2023-06-21
+Released: `@100mslive/hms-video-store@0.10.8`, `@100mslive/react-sdk@0.8.8`, `@100mslive/hls-player@0.1.8`
+
+### Added
+- A flag to disallow auto video unsubscription
```

Comment:
should we add this? this was added a fix for growthday

Reply:
there is HLS player fix as well, but it is on App side.

Reply:
yeah lets not have this for now as its not a sdk level update. Its an internal update